wiki/People/Featured_Artists/de.md

```diff
@@ -0,0 +1,130 @@
+---
+tags:
+  - FA
+---
+
+# Gesponserte Künstler
+
+**Gesponserte Künstler** (auch *Featured Artists* oder kurz ***FA*** genannt) sind Musikschaffende, die mit osu! zusammenarbeiten, um einige ihrer Songs für die Verwendung in [Beatmaps](/wiki/Beatmap) und als Hintergrundmusik in den Livestreams des [osu! World Cup](/wiki/Tournaments#osu!-world-cup) zur Verfügung zu stellen.
+
+Die [Auflistung der Featured Artists](https://osu.ppy.sh/beatmaps/artists) auf der Webseite präsentiert jeden einzelnen Künstler, der an dem Programm teilnimmt. Jede Künstlerseite enthält allgemeine Informationen über die Musik und eine Liste der Songs, die für die Verwendung in osu! lizenziert sind, zusammen mit vorkonfigurierten `.osz`-Dateien zur Erstellung von Beatmaps.
+
+Das [osu!-Team](/wiki/People/osu!_team) arbeitet manchmal mit Künstlern zusammen, um neue Exklusivtitel für osu! zu produzieren, genannt [osu! originals](/wiki/osu!_originals). Diese werden als solche im FA-Katalog gekennzeichnet.
+
+## Auswahl
+
+Neuzugänge zum Featured-Artist-Programm werden indirekt von der osu!-Community bestimmt. Wenn Mapper Beatmaps für einen Künstler entwerfen und ranken, spricht das osu!-Team den Künstler möglicherweise an, um auf das Featured-Artist-Programm aufmerksam zu machen. Gelegentlich werden Künstler jedoch basierend auf anderen Faktoren zum Featured-Artist-Programm eingeladen, wie beispielsweise ihre Präsenz in der Mapping-Community von osu!.
```

Comment:
I tried to avoid the verb `ranken` as much as possible, but it's not possible without loosing precision I think.

There is another occurence with `das/dem Ranking`.